src/org/zaproxy/zap/extension/pscan/PassiveScanThread.java

```diff
@@ -143,6 +146,10 @@ public void run() {
 						
 						for (PassiveScanner scanner : scannerList.list()) {
 							try {
+								if (View.isInitialised()) {
```

Comment:
This should be done outside the loop (the message is still being scanned here thus the count will not change), maybe after line 190?